lorien/InfiniteCanvas/InfiniteCanvas.gd

```diff
@@ -45,8 +45,16 @@ func _ready():
 	_camera.connect("position_changed", self, "_on_camera_moved")
 	_viewport.size = OS.window_size
 
+# -------------------------------------------------------------------------------------------------
+func _unhandled_key_input(event: InputEventKey) -> void:
+	_process_event(event)
```

Comment:
Not sure what happened here, I think my testing when creating the PR for this was not super clean.

This makes sure that key events are collected. They are not passed as regular `_gui_input`-events. So they need to be captures like this.